freqtrade/configuration/configuration.py

```diff
@@ -10,7 +10,7 @@
 
 from freqtrade import OperationalException, constants
 from freqtrade.configuration.check_exchange import check_exchange
-from freqtrade.configuration.create_datadir import create_datadir
+from freqtrade.configuration.folder_operations import create_datadir
```

Comment:
no folders in unixes... o_0

Reply:
i always wonder how you notice this - i had to think 3 times (and took the wrong option), since for me directory and folder is interchangeable.

Reply:
I was writing file system drivers some long time ago...
Additionally, I know from the times of Windows 3.1 that the folder term is only used in Windows GUI, not in CDE and DCE or POSIX calls...

Actually, in Unixes, everything is file, even directories... 😆